importer/BeBruNl.py

```diff
@@ -0,0 +1,131 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class BeBruNl(Monument):
+
+    def set_adm_location(self):
+        brussel_region = "Q240"
+        location = self.plaats
+        municip_dict = self.data_files["municipalities"]
+        if location:
+            location_match = [x for x in municip_dict if x[
+                "itemLabel"] == location]
+            if len(location_match) == 1:
+                self.add_statement("located_adm", location_match[0]["item"])
+        else:
+            self.add_statement("located_adm", brussel_region)
+
+    def set_heritage(self):
+        if self.has_non_empty_attribute("beschermd"):
+            prot_date = self.beschermd
+            try:
+                date_dict = utils.date_to_dict(prot_date, "%d/%m/%Y")
+                qualifier = {"start_time": {"time_value": date_dict}}
+                heritage = self.mapping["heritage"]["item"]
+                self.add_statement("heritage_status", heritage, qualifier)
+            except ValueError:
+                self.add_to_report("beschermd", self.beschermd)
+                return super().set_heritage()
+        else:
+            return super().set_heritage()
+
+    def set_address(self):
+        street_with_no = utils.remove_markup(self.adres.title()).rstrip(',')
+        whole_address = "{}, {}".format(street_with_no, self.plaats)
+        self.add_statement("located_street", whole_address)
+
+    def set_special_is(self):
+        objtype = self.objtype.lower()
+        types = self.data_files["type"]["mappings"]
+        special_types = [types[x]["items"]
```

Comment:
you use this mechanism so often (also in other classes) that it might be worth introudicing the following in your utils
```python
def get_matching_items_from_dict(value, dict_name):
    """
    Return all items in a dict for which the label matches the provided value.
    
    @param value: the value to match
    @param dict_name: the dict to look in
    """
    return [dict_name[x]["items"]
            for x in styles if x.lower() == value]
```

Reply:
Done, added this to utils and substituted in class.